bigquery/client.py

```diff
@@ -276,7 +276,7 @@ def _insert_job(self, body_object):
             body=body_object
         ).execute()
 
-    def query(self, query, max_results=None, timeout=0, dry_run=False, use_legacy_sql=None):
+    def query(self, query, max_results=None, timeout=0, dry_run=False, use_legacy_sql=None, external_udf_uris=[]):
```

Comment:
So I've fixed the existing write_to_table's default arg as well, in a force update.